src/Core/Common/NeighborhoodIteratorOnVectorImage/Documentation.rst

```diff
@@ -8,7 +8,7 @@ Neighborhood Iterator On Vector Image
 Synopsis
 --------
 
-NeighborhoodIterator on a VectorImage.
+Neighborhood Iterator on a VectorImage.
```

Comment:
Not sure if I'd change this, since `VectorImage` should also be split by the same token. I guess it is difficult to be consistent in absence of written rules, i.e. using the concepts rather than the class names. Unless the plan is to progressively change them as time permits, I'd leave this as it is.